src/app/admin/emails/emails.component.html

```diff
@@ -17,90 +17,185 @@
+          scope="col"
+        >
+          <span [hidden]="!hideFilters">Subject</span>
+          <div [hidden]="hideFilters">
+            <mat-form-field class="filter" floatLabel="never">
+              <mat-label>Subject</mat-label>
+              <input #filters matInput name="subject" (keyup)="applyFilter()" />
+            </mat-form-field>
+          </div>
+        </th>
+        <td
+          mat-cell
+          *matCellDef="let record"
+          style="text-align: left; word-wrap: break-word"
+        >
+          <span
+            ><strong>{{ record.subject }}</strong></span
+          >
+        </td>
+      </ng-container>
+      <ng-container matColumnDef="body">
+        <th
+          mat-header-cell
+          *matHeaderCellDef
+          mat-sort-header="body"
+          [disabled]="!hideFilters"
+          style="cursor: pointer; width: 40%"
+          columnName="body"
+          scope="col"
+        >
+          <span>Body</span>
+        </th>
+        <td
+          mat-cell
+          *matCellDef="let record"
+          style="text-align: left; word-wrap: break-word"
+        >
+          <mdm-more-description
+            [description]="record.body"
+          ></mdm-more-description>
+        </td>
+      </ng-container>
+      <ng-container matColumnDef="successfullySent">
+        <th
+          mat-header-cell
+          *matHeaderCellDef
+          mat-sort-header="successfullySent"
+          [disabled]="!hideFilters"
+          style="cursor: pointer; width: 10%"
+          columnName="successfullySent"
+          scope="col"
+        >
+          <span [hidden]="!hideFilters">Status</span>
+        </th>
+        <td mat-cell *matCellDef="let record" style="text-align: left">
+          <div>
+            <a *ngIf="record.successfullySent"> Successful </a>
+            <a
+              *ngIf="!record.successfullySent"
+              (click)="toggleMessage(record)"
+              style="text-decoration: none; cursor: pointer"
+            >
+              Failed
+            </a>
+            <div
+              *ngIf="!record.successfullySent && record.showFailure"
+              style="margin-top: 5px"
+            >
+              {{ record.failureReason }}
+            </div>
+            <button
+              mat-button
+              color="warn"
+              class="block"
+              (click)="toggleMessage(record)"
+              *ngIf="record.showFailure"
+            >
+              Hide details
+              <span class="fas fa-chevron-right"></span>
+            </button>
+          </div>
+        </td>
+      </ng-container>
+      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+      <tr mat-row *matRowDef="let record; columns: displayedColumns"></tr>
+    </table>
+  </div>
+  <div
+    class="bordered is-hidden pxy-2"
+    [ngClass]="{ block: !totalItemCount && !isLoadingResults }"
+  >
+    <p class="marginless">There are no Emails in this model.</p>
```

Comment:
I don't think this sentence makes sense in this table. Perhaps it should say "There are no emails to display." instead.